nmrfx-processor-gui/src/main/java/org/nmrfx/processor/gui/spectra/DatasetAttributes.java

```diff
@@ -928,7 +928,7 @@ public void getProjection(Dataset dataset, Vec specVec, int iDim) throws IOExcep
         ptC[0][0] = dataset.ppmToPoint(0, ppm0);
         ptC[0][1] = dataset.ppmToPoint(0, ppm1);
         specVec.resize(ptC[0][1] - ptC[0][0] + 1, dataset.getComplex_r(0));
-        dataset.readVectorFromDatasetFile(ptC, dimC, specVec);
+        dataset.readVectorFromDatasetFile(DatasetUtils.generateRawIndices(ptC, dataset.getComplex_r(0)), dimC, specVec);
```

Comment:
Dataset has both getComplex and getComplex_r ... they are very similar, I'm not sure which one to use?

Reply:
Should be getComplex here (though should work with both when not processing)

Reply:
What about the line above it (930) should that also be getComplex?

Reply:
changed both to getComplex